src/pages/ConnectWallet/index.tsx

```diff
@@ -190,12 +248,12 @@ const _ConnectWalletPage = ({
       !error &&
       (balance || balance === 0)
     ) {
-      setStatus(`${balance} ${TICKER_NAME} available`);
+      setStatus(`${balance} ${TICKER_NAME}`);
     } else if (walletConnected && error) {
       setStatus('Error');
     } else if (!networkAllowed) {
       setStatus(
-        `Please connect to ${
+        `Connect ${getWalletName(walletProvider)} to ${
```

Comment:
@samajammin @wackerow any idea how we set up this kinda string for translataions?